### PR TITLE
Expose TransformsInto* targetline colors to yaml. 

### PR DESCRIPTION
**Pull request by pchote**
Followup to #18483.
Depends on #18821.

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/Buildings/TransformsIntoTransforms.cs [hunk 1]
```diff
@@ -44,7 +43,7 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 			if (!order.Queued)
 				currentTransform.NextActivity?.Cancel(self);
 
-			currentTransform.Queue(new IssueOrderAfterTransform("DeployTransform", order.Target, Color.Green));
+			currentTransform.Queue(new IssueOrderAfterTransform("DeployTransform", order.Target));
```
Comment (Mailaender) [inline OpenRA.Mods.Common/Traits/Buildings/TransformsIntoTransforms.cs:46]: Why is this one not needed?

Comment (pchote) [reply OpenRA.Mods.Common/Traits/Buildings/TransformsIntoTransforms.cs:46]: Deploying happens at the actor's current location, so doesn't have a target line.

**Review by Mailaender — DISMISSED** [review]
Nice improvement for modders.

**Comment by reaperrr** [comment]
Needs rebase now.

**Comment by pchote** [comment]
Rebased.

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/20aa1625e5e6f23c4916fd582c7b5b20dc76618b...e1325ce8953e49376a71899848d97e24ecb9ad04)

**Pull request merged**